### ISSUE TITLE
Disable direct access grant by default

### ISSUE THREAD

**Issue by stianst** [keycloak/keycloak#15693]
### Description

New clients should not have `direct access grant` enabled by default.

### Motivation

Resource owner credential grant (direct grant in Keycloak) is not recommended neither by Keycloak or by the OAuth specifications as such it should not be enabled by default.

**Comment by lrozenblyum** [comment]
Hello. I assume this issue is a duplicate of the already implemented https://github.com/keycloak/keycloak/issues/30226? @stianst?

**Comment by stianst** [comment]
@lrozenblyum yes, that's correct. On the backend it was disabled by default, but the UI used to enable it by default
